app/_includes/foot.html

```diff
@@ -201,13 +201,6 @@
     },
     this
   );
-  var _gaq = [
-    ["_setAccount", "UA-32956520-1"],
-    ["_setDomainName", ".yeoman.io"],
-    ["_trackPageview"],
-    ["_trackPageLoadTime"]
-  ];
-  $script("https://www.google-analytics.com/ga.js");
   $script("https://apis.google.com/js/plusone.js");
```

Comment:
We might want to remove this too? https://support.google.com/chat/answer/14087009?hl=en, AFAIK G+ is dead

Reply:
Good catch. I just removed it.